src/renderer/components/+workloads-pods/pod-container-env/view.test.tsx

```diff
@@ -0,0 +1,277 @@
+/**
```

Comment:
All of these should be in behaviours. But I'm not sure if we have all bits and pieces ready to test `Details`-panels in the behaviours. So OK for now.